masonry/src/widget/prose.rs

```diff
@@ -275,11 +284,12 @@ impl Widget for Prose {
     }
 
     fn accessibility_role(&self) -> Role {
-        Role::Paragraph
+        Role::Document
     }
 
-    fn accessibility(&mut self, _ctx: &mut AccessCtx, node: &mut NodeBuilder) {
-        node.set_name(self.text().as_ref().to_string());
+    fn accessibility(&mut self, ctx: &mut AccessCtx, node: &mut NodeBuilder) {
+        node.set_read_only();
+        self.text_layout.accessibility(ctx.tree_update, node);
```

Comment:
Any reason the same code isn't used in `Label::accessibility`?

Reply:
Yes, we ought to do this for labels too. I'd like to wait and do that after I do an AccessKit refactor so we don't have to also set the name property on the label. I don't think that should block merging this PR.